app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: post) do |form| %>
+  <% if post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% post.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
```

Comment:
we can use ViewComponents, we have some defined for the form fields
something like this should do the trick (ping me on Slack if you need help with this)

`      <%= render(FieldComponent.new(:title, form: form, type: :text, placeholder: 'Title', required: true)) %>`